common/locales/en/settings.json

```diff
@@ -41,7 +41,7 @@
     "json": "(JSON)",
     "customDayStart": "Custom Day Start",
     "24HrClock": "24Hr Clock",
-    "clockInfo": "HabitRPG defaults to check and reset your Dailies at midnight each day. You can customize that here (Enter number between 0 and 24).",
+    "customDayStartInfo": "HabitRPG defaults to checking and resetting your Dailies at midnight each day. You can customize that here. Enter a number between 0 and 24. It is recommended that you first read <a href=\"http://habitrpg.wikia.com/wiki/Custom_Day_Start\" target=\"_blank\">this advice about how to change it</a>.",
```

Comment:
I like it, although we might want to include a cautionary sentence about
how to avoid double-cron (if that still occurs) directly in the warning
itself.

On Fri, Apr 3, 2015 at 4:01 AM, Matteo Pagliazzi <notifications@github.com>
wrote:

> In common/locales/en/settings.json
> <https://github.com/HabitRPG/habitrpg/pull/4965#discussion_r27724614>:
>
> > @@ -41,7 +41,7 @@
> >      "json": "(JSON)",
> >      "customDayStart": "Custom Day Start",
> >      "24HrClock": "24Hr Clock",
> > -    "clockInfo": "HabitRPG defaults to check and reset your Dailies at midnight each day. You can customize that here (Enter number between 0 and 24).",
> > +    "customDayStartInfo": "HabitRPG defaults to checking and resetting your Dailies at midnight each day. You can customize that here. Enter a number between 0 and 24. It is recommended that you first read <a href=\"http://habitrpg.wikia.com/wiki/Custom_Day_Start\" target=\"_blank\">this advice about how to change it</a>.",
>
> but it's really a 2 minutes thing, so feel free to ask if you need it
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/HabitRPG/habitrpg/pull/4965/files#r27724614>.
>